README.md

```diff
@@ -1,7 +1,25 @@
 ![Python application](https://github.com/BAMresearch/BayesianInference/workflows/Python%20application/badge.svg?branch=master)
 
 # Bayesian Inference
-Collection of algorithms and helper classes to efficiently solve inference problems with both variational Bayesian and sampling methods.
+
+`numpy`-based implementation of an analytical variational Bayes algorithm of
+
+> "Variational Bayesian inference for a nonlinear forward model." 
+> 
+> Chappell, Michael A., Adrian R. Groves, Brandon Whitcher, and Mark W. Woolrich. 
+> IEEE Transactions on Signal Processing 57, no. 1 (2008): 223-236.
+
+with an updated free energy equation to correctly capture the log evidence. 
+
+The recently published [authors implementation](https://github.com/physimals/vaby_avb) provides a set of models **g(θ)** that can be combined to a time-series-like dataset **y**, forming the model error **k(θ) = g(θ) - y**. The algorithm presented here is based directly on a user-defined model error **k(θ)** to allow a arbitrary combinations of custom forward models and the measured data.
+
```

Comment:
Ich würde das nicht so spezifisch machen.
Vielleicht eher so:
The algorithm is based on a user-defined model error to allow a arbitrary combination of custim forward models and measured data.

A recently puplished implementation by the group of Chappell based on tensorflow can be found [here](...).

Reply:
oder ganz weglassen?